backend/api/models/groups.js

```diff
@@ -3,7 +3,8 @@ const mongoose = require('mongoose');
 const groupSchema = new mongoose.Schema({
     name: {
         type: String,
-        required: true
+        required: true,
+        maxlength: 15
```

Comment:
32 na froncie można to przecież pozawijać? :>

Reply:
Done